apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+| transaction_timestamp       | String  | Timestamp when the transaction occurred. Ex. "2024-04-17T02:14:25.68771"                                                                             |
+| transaction_version         | bigint  | Blockchain version of the transaction. Ex. 2                                                                                                         |
+| type                        | String  | Type of the transaction, described by a Move entry function. Ex. "0x3::token::TokenStore"                                                            |
+
+### `current_fungible_asset_balances`
+
+_Has an aggregate view for summary data called `current_fungible_asset_balances_aggregate`_
+
+This tracks the asset balances of each account on-chain. It includes v1 token data.
+
+| Field                      | Type    | Description                                                                                                                       |
+| -------------------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| amount                     | bigint  | The amount of the asset owned. Ex. 2000                                                                                           |
+| asset_type                 | String  | The type of the asset, described by a Move resource. Ex. "0x1::aptos_coin::AptosCoin"                                             |
+| is_frozen                  | Boolean | Indicates whether the asset is frozen. Ex. False                                                                                  |
+| is_primary                 | Boolean | Indicates whether this is the primary balance of the owner. Ex. True                                                              |
+| last_transaction_timestamp | String  | Timestamp of the last transaction involving this balance. Ex. "2024-04-17T02:14:25.68771"                                         |
+| last_transaction_version   | bigint  | Blockchain version of the last transaction involving this balance. Ex. 30000000                                                   |
+| metadata                   | Object  | Use the [Hasura explorer](#indexer-api-reference) to see fields for `metadata` in `current_fungible_asset_balances`.              |
+| owner_address              | String  | This is an Aptos account address that owns the asset. Ex. "0xa815a9a09105973084bfc31530e7c8f002846787c2f0521e1e34dc144ad83b89"    |
+| storage_id                 | String  | Identifier for the storage associated with this balance. Ex. "0xa815a9a09105973084bfc31530e7c8f002846787c2f0521e1e34dc144ad83b89" |
+| token_standard             | String  | Aptos standard that the collection adheres to. Ex. "v1"                                                                           |
+
+## Aptos Naming Service (ANS)
+
+### `current_aptos_names`
```

Comment:
I may just be unfamiliar with how GraphQL gets used - do end users need the view construction? (If not, I think it may be a bit confusing to show it since they would still need to know what each field means to understand it).